libs/blocks/global-footer/global-footer.js

```diff
@@ -153,7 +153,9 @@ class Footer {
 
   loadIcons = async () => {
     const file = await fetch(`${base}/blocks/global-footer/icons.svg`);
-
+    if (!file.ok) {
+      throw new Error(`${file.statusText} url: ${file.url}`);
```

Comment:
I don't think throwing an error here is correct. Due to a network failure this can happen at any point and then we'd destroy the UX instead of gracefully failing (just not showing the icons)

### Before, load fails
Social icons are missing, but the footer looks "okay" from a consumer perspective
![Screenshot 2024-06-03 at 10 29 00](https://github.com/adobecom/milo/assets/39759830/160d94f9-a601-453a-bd89-a6819c1bef2a)

### After
Footer is non-existent // only an empty box
![Screenshot 2024-06-03 at 10 30 15](https://github.com/adobecom/milo/assets/39759830/706d72be-1c6d-4ec0-ac02-1394210a1559)

Reply:
Understood, yes, we can refrain from including these types of Lana logs.

Reply:
It's okay to log this if you see value in it for debugging purposes, I'd just not make the method fail inorder to log something.

Reply:
Added a log as error type warning instead of method fail by throwing error.